github_api.rb

```diff
@@ -0,0 +1,41 @@
+require "rest-client"
+require "json"
+
+# GithubRepoApi Class
+# This class is here for communicating with the Github API to retrieve information
+# about a specific repo.
+#
+# Arguments:
+# username - The username to authenticate to github as (ex: user1234)
+# password - The password to quthenticate to github with (ex: password1234)
+# repo - The repository in github that you'd like to be getting info for (ex: Invoca/web)
+class GithubRepoApi
+  def initialize(username, password, repo)
+    @github_url = "https://#{username}:#{password}!@api.github.com/repos/#{repo}"
+  end
+
+  # Returns a hash containing the base information
+  # for the repository including but not limited to:
+  #
+  # fork - Boolean indicating if the repo was forked
+  def information!
+    JSON.parse(RestClient.get(@github_url))
+  end
```

Comment:
Since you aren't mutating anything, there's no need for the bang (!) in the method name.
```suggestion
  def information
    JSON.parse(RestClient.get(@github_url))
  end
```

**Note:** it looks like you're only using this to get the value of `fork`. Consider renaming this function and making it more specific, like:
```suggestion
  def forked?
    JSON.parse(RestClient.get(@github_url))["fork"]
  end
```